File: src/main/scala/org/openapitools/server/model/GpsDTO.scala
package org.openapitools.server.model


/**
 * GPS-координаты широты и долготы. 
 *
 * @param latitude Широта. for example: ''null''
 * @param longitude Долгота. for example: ''null''
*/
final case class GpsDTO (
  latitude: Double,
  longitude: Double
)

File: ympa_fsharp_functions_server/src/model/GpsDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json

module GpsDTO =

  //#region GpsDTO

  [<CLIMutable>]
  type GpsDTO = {
    [<JsonProperty(PropertyName = "latitude")>]
    Latitude : decimal;
    [<JsonProperty(PropertyName = "longitude")>]
    Longitude : decimal;
  }

  //#endregion
